app/recurring_job.go

```diff
@@ -121,6 +130,15 @@ func recurringJob(c *cli.Context) (err error) {
 		return errors.Wrap(err, "failed to get JSON encoding for labels")
 	}
 
+	jobParameterMap := map[string]string{}
+	if recurringJob.Spec.Parameters != nil {
+		jobParameterMap = recurringJob.Spec.Parameters
+	}
+	parameterJSON, err := json.Marshal(jobParameterMap)
+	if err != nil {
+		return errors.Wrap(err, "failed to get JSON encoding for parameters")
+	}
```

Comment:
The JSON string is only for logging. Can we simplify it as https://go.dev/play/p/lxjH7awoAt9?

Reply:
fixed.